### PR TITLE
Pre release notes `0.26.0`

### PR DESCRIPTION
**Pull request by rmoyard**
**Description of the Change:**

Start working on the release notes before release candidate branch is created.

**Release notes**
* [x] Create sections
* [ ]  Organize the order of entries
* [ ] Shorten the examples
* [ ] Check that the examples execute correctly
* [x] All PRs in the changelog?
* [x] Double-check the contributors in the list (+ code reviewers)

### CODE REVIEW

File: doc/releases/changelog-dev.md [hunk 1]
```diff
@@ -4,78 +4,103 @@
...
+  tensor(0.80473785, requires_grad=True)
+  ```
+  
+  We will continue to add more and more support for qutrits in future releases!
 
-**Classical shadows**
+<h4>Classical shadows 👤</h4>
 
-* Added the `qml.classical_shadow` measurement process that can now be returned from QNodes.
+* Classical shadow measurements are now available to return from QNodes.
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:87]: Need PR links

File: doc/releases/changelog-dev.md [hunk 2]
```diff
@@ -86,37 +111,43 @@
...
+      [1., 1.], 
+      [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliX(0) @ qml.PauliX(1)]
+  ) 
+  >>> circuit(x, H), 
+  tensor(1.8486, requires_grad=True) 
+  >>> qml.grad(circuit)(x, H))
+  -0.4797000000000001
   ```
 
+<h4>Operator stuff (TODO: IMPROVE TITLE)</h4>
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:149]: I put some things in this subsection, but it feels a little all over the place.

Comment (isaacdevlugt) [reply doc/releases/changelog-dev.md:149]: Can't comment on it 😢 

line 165 (`qml.exp`):

Was this not a part of v0.25?

**Review by isaacdevlugt — CHANGES_REQUESTED** [review]
I will have more comments as I go through it more and more, but this is a decent place to start

**Comment by rmoyard** [comment]
Names, contributions and changelog entries number are checked.

File: doc/releases/changelog-dev.md (hunk 1 above)
Comment (rmoyard) [reply doc/releases/changelog-dev.md]: I added them.

File: doc/releases/changelog-dev.md [hunk 3]
```diff
@@ -209,31 +322,6 @@
...
-  True
-  ```
-
-<h3>Improvements</h3>
-
-* The new `qml.pow` provides a top-level constructor for raising operators to powers.
-  [(#3029)](https://github.com/PennyLaneAI/pennylane/pull/3029)
-
 * `qml.matrix` now can also compute the matrix of tapes/QNodes that contain multiple
   broadcasted operations, or non-broadcasted operations after broadcasted ones.
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:326]:
```suggestion
`qml.matrix` can now compute the matrix of tapes and QNodes that contain multiple
broadcasted operations or non-broadcasted operations after broadcasted ones.
```

File: doc/releases/changelog-dev.md [hunk 4]
```diff
@@ -257,50 +348,12 @@
...
-  ```pycon
-  >>> op = qml.PauliX(wires=0)
-  >>> qml.is_hermitian(op)
-  True
-  >>> op2 = qml.RX(0.54, wires=0)
-  >>> qml.is_hermitian(op2)
-  False
-  ```
-
 * Per default, counts returns only the outcomes observed in sampling. Optionally, specifying `qml.counts(all_outcomes=True)`
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:354]:
```suggestion
* By default, `qml.counts` only returns the outcomes observed in sampling. Optionally, specifying `qml.counts(all_outcomes=True)`
```

File: doc/releases/changelog-dev.md [hunk 5]
```diff
@@ -372,12 +394,11 @@
...
+* `qml.Barrier` with `only_visual=True` now simplifies via `op.simplify()` to the identity
   or a product of identities.
   [(#3016)](https://github.com/PennyLaneAI/pennylane/pull/3016)
 
-* `__repr__` and `label` methods are more correct and meaningful for Operators with an arithmetic
-  depth greater than 0. The `__repr__` for `Controlled` show `control_wires` instead of `wires`.
+* More correct and intuitive outputs for printing some operators have been added.
   [(#3013)](https://github.com/PennyLaneAI/pennylane/pull/3013)
 
 * Use `Operator.hash` instead of `Operator.matrix` to cache the eigendecomposition results in `Prod` and
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:404]: Confused by this 🤔

**Review by isaacdevlugt — CHANGES_REQUESTED** [review]

File: doc/releases/changelog-dev.md [hunk 6]
```diff
@@ -168,85 +307,15 @@
...
-  [(#2922)](https://github.com/PennyLaneAI/pennylane/pull/2922)
-
-  ```pycon
-  >>> adj_op = qml.adjoint(qml.RX(1, 0))
-  >>> qml.simplify(adj_op)
-  RX(-1, wires=[0])
-  ```
-
-* Added `sparse_matrix()` support for single qubit observables
+* Added `sparse_matrix()` support for single qubit observables.
```
[the remark this answers is not in the document]
Comment (rmoyard) [reply doc/releases/changelog-dev.md:379]: Maybe it is simply that you can call `op.sparse_matrix()`.

File: doc/releases/changelog-dev.md [hunk 7]
```diff
@@ -168,85 +307,15 @@
...
-  True
-  >>> op2 = qml.RX(0.54, wires=0)
-  >>> qml.is_hermitian(op2)
-  False
-  ```
-
 * Internal use of in-place inversion is eliminated in preparation for its deprecation.
   [(#2965)](https://github.com/PennyLaneAI/pennylane/pull/2965)
 
 * `qml.is_commuting` is moved to `pennylane/ops/functions` from `pennylane/transforms/commutation_dag.py`.
```
[the remark this answers is not in the document]
Comment (rmoyard) [reply doc/releases/changelog-dev.md:402]: It should not change anything for users, expect that the function was moved to another folder.

File: doc/releases/changelog-dev.md (hunk 5 above)
Comment (rmoyard) [reply doc/releases/changelog-dev.md:432]: Here they are caching the results of the matrix of sum/prod of operators, this is saved not with the matrix but with the operator hash now. Users don't use it directly.

File: doc/releases/changelog-dev.md [hunk 8]
```diff
@@ -284,7 +353,6 @@
 * The default `execute` method for the `QubitDevice` base class now calls `self.statistics`
   with an additional keyword argument `circuit`, which represents the quantum tape
   being executed.
-
   Any device that overrides `statistics` should edit the signature of the method to include
   the new `circuit` keyword argument.
   [(#2820)](https://github.com/PennyLaneAI/pennylane/pull/2820)
```
[the remark this answers is not in the document]
Comment (rmoyard) [reply doc/releases/changelog-dev.md:459]: The function statistics (_qubit_device) is changed where a keyword is added, that means that you can change that if you create a custom device.

File: doc/releases/changelog-dev.md [hunk 9]
```diff
@@ -168,85 +307,15 @@
...
-  ```pycon
-  >>> op = qml.PauliX(wires=0)
-  >>> qml.is_hermitian(op)
-  True
-  >>> op2 = qml.RX(0.54, wires=0)
-  >>> qml.is_hermitian(op2)
-  False
-  ```
-
 * Internal use of in-place inversion is eliminated in preparation for its deprecation.
```
[the remark this answers is not in the document]
Comment (rmoyard) [reply doc/releases/changelog-dev.md]: Inplace inversion is replace by using adjoint.

[the remark this answers is not in the document]
Comment (rmoyard) [reply doc/releases/changelog-dev.md]: Check deprecation section for more details

File: doc/releases/changelog-dev.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (isaacdevlugt) [reply doc/releases/changelog-dev.md:402]: Okay I think this can just be removed from the changelog then!

[the remark this answers is not in the document]
Comment (rmoyard) [reply doc/releases/changelog-dev.md:402]: I think it is better to keep a history of this kind of changes for devs as well, imo it is better to leave it there.

File: doc/releases/changelog-dev.md [hunk 10]
```diff
@@ -23,179 +26,289 @@
...
+  ... ])
+  >>> op = qml.AmplitudeEmbedding(features, wires=[1, 5, 2])
+  >>> op.batch_size
+  3
   ```
 
-* `expand_matrix()` method now allows the sparse matrix representation of an operator to be extended to
+  An exception is `BasisEmbedding`, which is not broadcastable.
+
+<h3>Improvements</h3>
```
Comment (rmoyard) [inline doc/releases/changelog-dev.md:309]:
```suggestion
```

**Review by isaacdevlugt — APPROVED** [review]
LGTM
